libs/state/src/lib/inject.ts

```diff
@@ -0,0 +1,57 @@
+import { ChangeDetectorRef, ViewRef, inject } from '@angular/core';
+import { AccumulationFn } from '@rx-angular/state/selections';
+import { Observable } from 'rxjs';
+import { RxState } from './rx-state.service';
+
+export type RxStateFeature<State extends object> = (
+  rxState: RxState<State>
+) => void;
+
+export type RxStateFeatures<State extends object> = RxStateFeature<State>[];
+
+/**
+ * @todo: see how to infer generic type from accumulator to accumulatorFn.
+ */
+export function accumulator<State extends object>(
+  accumulatorFn: AccumulationFn
+) {
+  return (rxState: RxState<State>) => rxState.setAccumulator(accumulatorFn);
+}
```

Comment:
as far as I enjoy the shortness of the API, i'm concerned we will run into problems when we call the feature factories like this. Maybe we should stick to the angular pattern and call it `withAccumulator` instead? Same goes for `initialState` and others.

Maybe before even releasing those functions we should go for a public RFC?

Reply:
I agree, it's nice to have a prefix for those composition functions in order to reduce the confusion with other APIs.

Not sure about the need for an RFC, I'm afraid that we just lose time and effort but do not gather the feedback we need in the end. What about discussing it publicly here and avoiding side discussions?

Marking the APIs as experimental would allow us to test them, get feedback and eventually break them if we want to.

Reply:
how can we mark single APIs as experimental? You want to introduce a new subpackage for it?

Reply:
agree, let's discuss it here

Reply:
> how can we mark single APIs as experimental? You want to introduce a new subpackage for it?

Like Angular, marking them with a symbol and/or using JSDocs. We can also introduce a subpackage but I think the first way makes more sense for this case.

Reply:
v16 releases this week, we can just go for `DestroyRef` :)

Reply:
@hoebbelsB Even if we use DestroyRef, I would suggest marking all those new APIs as experimental.

Reply:
I'll rebase once #1567 is merged.